lib/zinnia.js

```diff
@@ -43,6 +43,22 @@ export async function start ({
   onActivity,
   onMetrics
 }) {
+  const startAgain = () => {
+    if (childProcess.exitCode == null && childProcess.signalCode == null) {
```

Comment:
Do you think this can happen? This is after the child process has emitted `close`, I don't see how this condition could ever be true